src/main/java/com/influxdb/v3/client/internal/VectorSchemaRootConverter.java

```diff
@@ -105,6 +105,32 @@ PointValues toPointValues(final int rowNumber,
         return p;
     }
 
+    private void setFieldWithMetaType(final PointValues p,
+                                      final String name,
+                                      final Object value,
+                                      final String metaType) {
+        if (value == null) {
+            return;
+        }
+
+        switch (metaType) {
+            case "iox::column_type::field::integer":
+            case "iox::column_type::field::uinteger":
+                p.setIntegerField(name, (Long) value);
+                break;
+            case "iox::column_type::field::float":
+                p.setFloatField(name, (Double) value);
+                break;
+            case "iox::column_type::field::string":
+                p.setStringField(name, (String) value);
+                break;
+            case "iox::column_type::field::boolean":
+                p.setBooleanField(name, (Boolean) value);
+                break;
+            default:
```

Comment:
There should be the implementation for `default` because metadata can be not defined.